CONTRIBUTING.md

```diff
@@ -8,10 +8,6 @@ _As contributors and maintainers of this project, and in the interest of fosteri
 
 We have full documentation on how to get started contributing here:
 
-<!---
```

Comment:
I prefer to do few larg PRs, rather then many small. Should have done in another commit.
Readme update replaces explanation how to run the controller by copying commands with `make run`. 
I changed binary name from `manager` to `node-ipam-controller` because `manager` does not convey any message what the binary manages.

Reply:
then better squash all commits, if I go commit by commit I can see things moving in one direction and later be reverted

Reply:
Usually, I try to do meaningful small commits, then squash&merge after review.
I should have done this here too, i.e. changing binary name as a separate commit, bumping chart version, updating readme. Sorry for the confusion.

Reply:
heh, no worries, I do the same, is just that these multiple commits are part of your development process, and it seems you found some wrong assumptions doing that ... that is not interesting for the reviewer, that expects the multiple commits to be reviewable independently